src/Compilers/CSharp/Test/Syntax/Parsing/FuzzTesting.cs

```diff
@@ -78,5 +78,13 @@ public void c024928()
 ";
             var tree = SyntaxFactory.ParseSyntaxTree(test);
         }
+
+        [WorkItem(2771, "https://github.com/dotnet/roslyn/issues/2771")]
+        [Fact(Skip = "The test input is not under version contol")]
```

Comment:
I'm sure we can fabricate a test that exercises the new code. How about using mscorlib.dll as the input?